src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -479,12 +495,51 @@ public void InsertCode(string text)
                 }
                 else
                 {
+                    InsertText(text);
+                }
+            }
+
+            private void InsertText(string text)
+            {
+                using (var transaction = UndoHistory_TestOnly.CreateTransaction(InteractiveWindowResources.TypeChar))
```

Comment:
This is just a bad name, it is meant to be used only for test purpose *externally*, which is actually done by `Undo_TestOnly` and `Redo_TestOnly` methods in `InteractiveWindow`. So I'm removing 'TestOnly' part here.